### PR TITLE
Updates text

### PR DESCRIPTION
**Pull request by Anudeepcse**
TL;DR .....

##### completed:
- commits
- created a branch

#### To do
- [ ] merge

### CODE REVIEW

File: _posts/0000-01-09-Anudeepcse.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+IPL is the best cricket league ?
```
Comment (Anudeepcse) [inline _posts/0000-01-09-Anudeepcse.md:6]: This is really good, but what about the EPL?

Comment (rajeshrajanala) [inline _posts/0000-01-09-Anudeepcse.md:6]: ICL is the best cricket league

File: _posts/0000-01-09-Anudeepcse.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (rajeshrajanala) [inline _posts/0000-01-09-Anudeepcse.md:6]: Its looks good

File: _posts/0000-01-09-Anudeepcse.md (hunk 1 above)
Comment (Naseer2311) [inline _posts/0000-01-09-Anudeepcse.md:6]: IPL is best but T20 World cup is great
```suggestion
IPL is the best cricket league ?
```

**Review by rajeshrajanala — CHANGES_REQUESTED** [review]

**Review by Naseer2311 — APPROVED** [review]

File: _posts/0000-01-09-Anudeepcse.md (hunk 1 above)
Comment (cs07886) [inline _posts/0000-01-09-Anudeepcse.md:6]: Nice!

**Review by cs07886 — COMMENTED** [review]
Nice

**Review by cs07886 — APPROVED** [review]

File: _posts/0000-01-09-Anudeepcse.md (hunk 2 above)
Comment (gusshawstewart) [inline _posts/0000-01-09-Anudeepcse.md:6]: looks good 😄

**Review by gusshawstewart — APPROVED** [review]

**Pull request closed without merging**